app/javascript/elm/src/Main.elm

```diff
@@ -988,38 +987,34 @@ viewCrowdMapOptions isCrowdMapOn crowdMapResolution selectedSession tooltipIcon
         ]
 
 
-viewCrowdMapCheckBox : Bool -> Path -> Html Msg
-viewCrowdMapCheckBox isCrowdMapOn tooltipIcon =
+viewCrowdMapToggle : Bool -> Path -> Html Msg
+viewCrowdMapToggle isCrowdMapOn tooltipIcon =
     div [ class "filters__toggle-group" ]
-        [ input
-            [ id "checkbox-crowd-map"
-            , type_ "checkbox"
-            , checked isCrowdMapOn
-            , Events.onClick ToggleCrowdMap
-            ]
-            []
-        , label [ for "checkbox-crowd-map" ] [ text "Crowd Map" ]
+        [ label [] [ text "CrowdMap:" ]
+        , viewToggleButton "off" (not isCrowdMapOn) ToggleCrowdMap
+        , viewToggleButton "on" isCrowdMapOn ToggleCrowdMap
         , Tooltip.view Tooltip.crowdMap tooltipIcon
         ]
 
 
 viewCrowdMapSlider : String -> Html Msg
 viewCrowdMapSlider resolution =
     div [ id "crowd-map-slider" ]
-        [ p []
-            [ text "Resolution" ]
-        , div []
-            [ input
+        [ label [] [ text <| "grid cell size: " ++ (String.fromInt <| 51 - (Maybe.withDefault 20 <| String.toInt resolution)) ]
```

Comment:
I'm not sure what would happen, but why a different default on line 1010?

Reply:
this will 99% never show but I changed the random number to 26 which maps to the default 25 resolution. thx!